pkg/auth/auth_handler.go

```diff
@@ -39,6 +42,7 @@ func NewAuthHandler(cfg Config, ocmCLient *ocm.Client, log logrus.FieldLogger) *
 		utils:      NewAuthUtils(cfg.JwkCert, cfg.JwkCertURL),
 		client:     ocmCLient,
 		log:        log,
+		Cache:      cache.New(1*time.Minute, 30*time.Minute),
```

Comment:
Timeout should probably be part of `Config`

Reply:
I will open a Jira to handle it in a different PR